packages/beacon-node/src/network/reqresp/handlers/dataColumnSidecarsByRange.ts

```diff
@@ -12,9 +12,16 @@ export async function* onDataColumnSidecarsByRange(
   db: IBeaconDb
 ): AsyncIterable<ResponseOutgoing> {
   // Non-finalized range of columns
-  const {startSlot, count, columns} = validateDataColumnSidecarsByRangeRequest(request);
+  const {startSlot, count, columns: requestedColumns} = validateDataColumnSidecarsByRangeRequest(request);
   const endSlot = startSlot + count;
 
+  if (requestedColumns.length === 0) {
+    throw new ResponseError(RespStatus.INVALID_REQUEST, "dataColumnSidecar requested without column indices");
+  }
+
+  const columns = requestedColumns.filter((c) => chain.custodyConfig.custodyColumns.includes(c));
+  if (columns.length === 0) return;
```

Comment:
```suggestion
  if (requestedColumns.length === 0) {
    throw new ResponseError(RespStatus.INVALID_REQUEST, "dataColumnSidecar requested without column indices");
  }

  const columns = requestedColumns.filter((c) => chain.custodyConfig.custodyColumns.includes(c));
  if (columns.length === 0)  {
    throw new ResponseError(RespStatus.INVALID_REQUEST, "did not request column indices that are custodied");
  }
```

Reply:
I like the erroring here.  May also want to error if they do not ask for anything we should have in addition to empty requests